src/Markdowser/ViewModels/MainWindowViewModel.cs

```diff
@@ -196,50 +199,57 @@ private static bool IsValidHttpUri(string uriString, [NotNullWhen(true)] out Uri
         return Uri.TryCreate(uriString, UriKind.Absolute, out uri) && (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps);
     }
 
-    private void FetchUrl()
+private void FetchUrl()
```

Comment:
Wouldn't it be better to add a `bool` parameter that enables and disables the cache to the `Fetch` method, instead of having almost the same method twice? 

```suggestion
private void FetchUrl(bool useCache = true)
```